### PR TITLE
Check length of strParams and don't use it as a condition

### PR DESCRIPTION
**Pull request by olivier-monaco**
Issue #3274

### CODE REVIEW

**Comment by olivier-monaco** [comment]
Better code on #3280

File: phalcon/mvc/router.zep [hunk 1]
```diff
@@ -574,7 +574,7 @@ class Router implements RouterInterface
 			 */
 			if fetch paramsStr, parts["params"] {
 				let strParams = substr(paramsStr, 1);
-				if strParams {
+				if strlen(strParams) > 0 {
```
Comment (sergeyklay) [inline phalcon/mvc/router.zep]: or `strParams->length()` :)

Comment (olivier-monaco) [inline phalcon/mvc/router.zep]: But substr will return "false" when paramsStr is only 1 character length. So pull request #3280 is better...

Comment (sergeyklay) [inline phalcon/mvc/router.zep]: :+1:

**Pull request closed without merging**